packages/flutter/lib/src/rendering/view.dart

```diff
@@ -300,7 +307,7 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
       // bottom drawn pixel is at 1919 position.
       bounds.bottom - 1.0 - _view.padding.bottom / 2.0,
     );
-    final SystemUiOverlayStyle? upperOverlayStyle = layer!.find<SystemUiOverlayStyle>(top);
+    final SystemUiOverlayStyle? upperOverlayStyle = layer!.find<SystemUiOverlayStyle>(top) ?? layer!.find<SystemUiOverlayStyle>(center);
```

Comment:
Why are we just "randomly" choosing to do this in the center if we cannot find anything at the top?

Reply:
because when we use custom title-bar as widget in app, "top" always look for title-bar itself,
as you mentioned, will be better if we can customize where the "upperOverlayStyle" lookup from outside of framework, but I can't find anything for do that. 
more detail about why I make this change 👇🏼
#125830

Reply:
This doesn't explain why center is the right place to look instead of top.

Reply:
We probably need to re-think how this needs to work for Desktop instead of slapping on a bandage like this.